homeassistant/components/climate/ecobee.py

```diff
@@ -269,6 +269,7 @@ def device_state_attributes(self):
             "fan": self.fan,
             "mode": self.mode,
             "operation": operation,
+            ATTR_OPERATION_LIST: self._operation_list,
```

Comment:
Could you please explain what you are trying to do here? The operation_list is not something that we would want to edit in the user interface. It is defined for the device and is unchangeable.